src/main/resources/application.properties

```diff
@@ -1 +1,7 @@
-
+spring.jpa.database=postgresql
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.properties.hibernate.auto_quote_keyword=true
```

Comment:
```
spring.jpa.database = org.hibernate.dialect.PostgreSQLDialect
```

Reply:
I tried above property but it didn't work. And seems it's default `PostgreSQL95Dialect` even without dialect specify:
```
2021-01-29 17:01:33.852  INFO 66144 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2021-01-29 17:01:33.863  INFO 66144 --- [           main] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.PostgreSQL95Dialect
2021-01-29 17:01:34.337  INFO 66144 --- [           main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
```